### PR TITLE
chore(release): v2.143.1

### PR DESCRIPTION
**Pull request by shikha372**
See [CHANGELOG](https://github.com/aws/aws-cdk/pull/30377/files#diff-9e3d5c5da890b65a67bf632f93416744d5b76f6eb239e42ab714c7768429ff99)

### CODE REVIEW

File: CHANGELOG.v2.alpha.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [2.143.1-alpha.0](https://github.com/aws/aws-cdk/compare/v2.143.0-alpha.0...v2.143.1-alpha.0) (2024-05-30)
```
Comment (moelasmar) [inline CHANGELOG.v2.alpha.md:5]: Please make it `##` instead of `###`

File: CHANGELOG.v2.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [2.143.1](https://github.com/aws/aws-cdk/compare/v2.143.0...v2.143.1) (2024-05-30)
```
Comment (moelasmar) [inline CHANGELOG.v2.md:5]: also here,  make it `##` instead of `###`

File: CHANGELOG.v2.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
...
+
+
+### Reverts
+
+* "fix(ses-actions): permissions too wide for S3 action" ([#30375](https://github.com/aws/aws-cdk/issues/30375)) ([6c716c6](https://github.com/aws/aws-cdk/commit/6c716c68ec2a222a1262577942ffde42002d2f44))
```
Comment (moelasmar) [inline CHANGELOG.v2.md:10]: remove the double quotes around the title

**Review by moelasmar — COMMENTED** [review]
added some minor comments

File: CHANGELOG.v2.alpha.md (hunk 1 above)
Comment (shikha372) [inline CHANGELOG.v2.alpha.md:5]:
```suggestion
## [2.143.1-alpha.0](https://github.com/aws/aws-cdk/compare/v2.143.0-alpha.0...v2.143.1-alpha.0) (2024-05-30)
```

File: CHANGELOG.v2.md (hunk 2 above)
Comment (shikha372) [inline CHANGELOG.v2.md:5]:
```suggestion
## [2.143.1](https://github.com/aws/aws-cdk/compare/v2.143.0...v2.143.1) (2024-05-30)
```

File: CHANGELOG.v2.md (hunk 3 above)
Comment (shikha372) [inline CHANGELOG.v2.md:10]:
```suggestion
* fix(ses-actions): permissions too wide for S3 action ([#30375](https://github.com/aws/aws-cdk/issues/30375)) ([6c716c6](https://github.com/aws/aws-cdk/commit/6c716c68ec2a222a1262577942ffde42002d2f44))
```

**Review by shikha372 — COMMENTED** [review]
Fixing changelog

**Review by moelasmar — APPROVED** [review]

**Review by scanlonp — CHANGES_REQUESTED** [review]
These commits are not squashed. Please consolidate the changelog edits into 1 chore commit.

**Pull request merged**